### PR TITLE
Update master php testing versions

### PR DESCRIPTION
**Pull request by nickvergessen**
Signed-off-by: Joas Schilling <coding@schilljs.com>

### CODE REVIEW

**Review by skjnldsv — APPROVED** [review]

File: .github/workflows/phpunit.yml [hunk 1]
```diff
@@ -87,7 +87,7 @@ jobs:
       # do not stop on another job's failure
       fail-fast: false
       matrix:
-        php-versions: ['7.3', '7.4']
+        php-versions: ['7.4', '8.0']
```
Comment (nickvergessen) [inline .github/workflows/phpunit.yml:90]: @vitormattos we need to update phpunit it seems?

Comment (vitormattos) [reply .github/workflows/phpunit.yml:90]: Is hardcoded on row 58, we can bump to `9.5.13`

**Comment by vitormattos** [comment]
@nickvergessen we will do backport to stable branches?

File: .github/workflows/phpunit.yml [hunk 2]
```diff
@@ -87,7 +87,7 @@ jobs:
       # do not stop on another job's failure
       fail-fast: false
       matrix:
-        php-versions: ['7.4', '8.0']
+        php-versions: ['7.3', '7.4']
```
Comment (nickvergessen) [inline .github/workflows/phpunit.yml:90]: you reverted the actual sense of the PR :D

**Comment by nickvergessen** [comment]
> we will do backport to stable branches?

No, they still support 7.3?

**Pull request merged**